extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -388,7 +388,7 @@
 					clearInterval(interval);
 					return;
 				}
-				_t.image.setAttribute("src", _t.url + Math.random().toString().slice(2));
+				_t.image.setAttribute("src", _t.url + Date.now());
```

Comment:
On this line and on line 378, you are still appending a number to the URL which is a bit blind to how the URL is formatted.  The [fix for ClassicUI](https://github.com/eclipse/smarthome/pull/1106/files#diff-ee0239b05dcc713c6860244dfd4d8688R76) instead replaces just the `t` parameter in the URL, which seems safer and more reliable.  Something like

```javascript
_t.image.setAttribute("src", _t.url.replace( /&t=\d+/g , '&t=' + new Date().getTime()));
```